### DISCUSSION TITLE
Google奧運廣告翻車！AI創意為何變公關災難？

### DISCUSSION THREAD

**Discussion by Tina-Peng-1566** [1101-GS4719-C/Python-Community-GS4719-C#2061] [General]
https://www.bnext.com.tw/article/79988/olympics-google-gemini-ad-controversy
Google奧運廣告翻車！Gemini代筆粉絲信踩雷，AI創意為何變公關災難？

AI很厲害、方便，但一定要用在對的、改該用的地方。
我認為，一個人會想寫信、寫卡片給對方，是因為發自內心的欣賞感謝或喜歡，若都由AI來寫，不就失去最原本的意義了嗎？
而且，AI再怎麼寫，都會有種隔閡、生疏感，尤其現在常會加一些「啊啊啊」、「額...」、「咳咳」之類的語助詞，甚至看起來有點語無倫次，但這才是真正人類會「順應時代潮流」寫出來的內容呀!
Apple的廣告就讓我感覺更驚悚了，等於是直接摧毀輾壓， 說未來不需要樂器、遊戲機、玩偶等這些實體的童年玩具，而是直接用iPad來取代。但人類會喜歡這些東西，不就是因為它有特殊的質感、造型等，會讓一個人對這些物品產生不同的情感和意義嗎？

有沒有什麼物品或事情，是你認為絕對不能由來AI取代或代理的嗎?
歡迎在下方留言分享你的想法~

**Comment by kingsophy** [comment]
對我來說，我認為法院是絕對不能由來AI取代或代理的。因為在一些涉及生命、權利和正義的情境中，人類之間的倫理和道德判斷是不可或缺的，法官的這些決策通常涉及複雜的價值觀考量，而非是單純的邏輯推理而已。如果使用AI來進行決策的話可能會忽略個案的特殊性，缺乏人的同理心，從而導致不公平的結果。

**Comment by CtyceT** [comment]
我認為心理學家或心理諮商師是無法被AI取代的。他們除了需要專業的知識以外，還需要身為人類才有的同理心、分析能力、以及洞察力。和他人產生互相信任的關係也是人類才能夠擁有的。

**Comment by ds0w0** [comment]
我認為保母與幼教員是無法被AI取代的。這些職業不僅需要專業知識，還涉及情感交流、關愛和對孩子的細微觀察，這些是目前AI無法完全做到的。孩子的成長需要真實的人際互動和情感支持，而這些都來自於與人的接觸與相處。AI或許可以輔助，但無法代替人類在這些角色中所扮演的重要角色。

**Comment by hunteryang0108** [comment]
我認為任何事物的發展都是建立在過去的事物之上的，因此不應該有完全取代過去技術的想法，Apple及Google的廣告在這方面的處理就沒有做得很好。在AI輔助我們進行工作的同時，我們仍應該持續思考人的本質是什麼，以及我們進行工作的初心是什麼。
